src/getdata.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable  prefer-template, prefer-promise-reject-errors, no-alert */
+
+import render from './render';
+
+async function getImage(input, info) {
+  const url = 'https://api.unsplash.com/photos/random?orientation=landscape&count=1&client_id=7XimdnnMOSmoeVFofk7rEM4_ApTXYE83u8ZAAlOfPZU&query=' + input;
+  const response = await fetch(url);
+  const data = await response.json();
+  render(data[0].urls.regular, info);
+}
+
+async function fetchData(input) {
+  const url = 'https://api.openweathermap.org/data/2.5/weather?q=' + input + ',&APPID=be7a948e30f210ef85214d3ba4b94273';
+  const response = await fetch(url);
+  const data = await response.json();
+  return new Promise((resolve, reject) => {
+    if (data.cod === '404') {
+      reject('No valid city');
+    } else {
+      resolve(data);
+    }
+  });
+}
+
+async function getData(input) {
+  try {
+    const request = await fetchData(input);
+    getImage(request.weather[0].main, request);
+  } catch (error) {
+    alert(error);
```

Comment:
- [ ] IMPORTANT - Please avoid using Window.alert in your code. Kindly remove this and all other cases where you added it. Handle errors in a better way. Window.alerts are ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/))